support/metrics/metricsTracker.go

```diff
@@ -105,6 +108,14 @@ type eventWrapper struct {
 	Events []event `json:"events"`
 }
 
+// UpdateLoopResult contains the results of the orderbook update.
+type UpdateLoopResult struct {
```

Comment:
structure of the struct looks right.
this should be accessible by the `update()` function as it's the result of that function. should live in `trader.go`
(see review about trade metrics handler mentioning that we should move this file metricsTracker.go to the plugins package)

Reply:
Even if we move `metricsTracker` to plugins, this struct cannot live in `trader.go`. That creates an import cycle, since `trader.go` already imports `plugins`. So, I think this struct has to live in `plugins/metricsTracker` (or somewhere else) to avoid cycles.